src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
> Having a minimized content attribute seems wrong. What does it even mean?

A minimized `content` here is just less code and HTML5 w3c compliant; ultimately this `meta` tag is used as a dummy element to verify that a CSS class properly applies. 

> the content should be XHTML-compliant. The minimized choice for this attribute goes against that. (And I'm not sure minimization is legal for anything except Boolean attributes.)


There's a lot of things about this `<meta>` tag that is non-XHTML compliant. For instance it can't have a `class` attribute or even exist in the `<body>`, sadly, we do both. I'm open to other "dummy" elements but `meta` tags/HTML5 compliance seem to be the most correct (purely for metadata) / consistent (been using HTML5 compliance as a baseline) approach.

> Separately where is the property attribute defined? Not mentioned in http://www.w3.org/TR/html5/document-metadata.html#the-meta-element

The `property` attribute is an open graph protocol attribute which w3c seems to validate for yet is rarely used or consumed. It's rarity is valuable to us since the tag is used in a "dummy" manner; the spec is [here](http://ogp.me/).